File: pset6/caesar.py
from cs50 import*
if(len(sys.argv)>=3 or len(sys.argv)==1):
        print(1)
else:        
    k = int(sys.argv[1])
    print("plaintext: ",end="")
    s=get_string()
    st=""
    for c in s:
        if(97<=ord(c) and ord(c)<=122):
            st=st+chr((ord(c)-97+k)%26+97)
        elif(65<=ord(c) and ord(c)<=90):
            st=st+chr((ord(c)-65+k)%26+65)
        else:
            st=st+c
    print("ciphertext: "+st)

File: pset2/caesar/caesar.c
#include<cs50.h>
#include<stdio.h>
int main(int argc , string argv[])
{
    int k,i=0;
    if(argc >=3 || argc==1)
    {
        printf("%d",1);
        return 1;
    }
    k = atoi(argv[1]);
    printf("plaintext: ");
    string s=get_string();
    while(s[i])
    {
        if('a'<=s[i]&&s[i]<='z')
        {
            s[i]=(s[i]-'a'+k)%26+'a';
        }
        if('A'<=s[i]&&s[i]<='Z')
        {
            s[i]=(s[i]-'A'+k)%26+'A';
        }
        i++;
    }
    printf("ciphertext: %s\n",s);
}